cmd/root.go

```diff
@@ -88,7 +130,11 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 	imgid = fmt.Sprintf("%s@%s", imagedata.Name, imagedata.Digest)
 	glog.Infof("Resolved to: %s", imgid)
 
-	if previousPivot == imgid {
+	targetMatched, err := compareOSImageURL(previousPivot, imgid)
+	if err != nil {
+		glog.Fatalf("%v", err)
```

Comment:
Minor: we pretty liberally use `glog.Fatalf` everywhere in this codebase. And since we're not actually adding any context here, I'd say just drop error-handling and Fatalf directly in `getRefDigest()`.

Reply:
It feels easier to keep the code looking similar to the MCD code - why change it unnecessarily, plus while I get the policy of using `Fatalf` where it's convenient, why change working code to stop using "proper" error handling?

Reply:
> why change working code to stop using "proper" error handling?

To keep it consistent with the rest of the whole codebase. Not strongly opposed though, just seems out of place. Keeping in sync with the MCD version is a good point though. Maybe add a comment about that?